back/spec/acceptance/static_pages_spec.rb

```diff
@@ -63,20 +63,20 @@
     post 'web_api/v1/static_pages' do
       with_options scope: :static_page do
         parameter :title_multiloc, 'The title of the static page, as a multiloc string', required: true
-        parameter :body_multiloc, 'The content of the static page, as a multiloc HTML string', required: true
+        parameter :top_info_section_multiloc, 'The content of the static page, as a multiloc HTML string', required: true
```

Comment:
Did you consider documenting more parameters as with `back/spec/acceptance/home_pages_spec.rb`?

Reply:
Probably a good idea. I'll do this tomorrow morning.

Reply:
In progress...

Reply:
Done.